### PR TITLE
fix(account ui adjustments)

### LINKED ISSUE #1108: Account UI Design
Image

Figma file reference: https://www.figma.com/file/6JdMdepU9BIH6gSvxAfOjW/Accounts-Revamp?node-id=162%3A16830

Changes

- Breadcrumb design
- Navigation Design (dont change the current links or elements in the navigation) just the design
- Fixed scrolling changes
- Instances sidebar changes

### LINKED ISSUE #1109: Add view all usage button
Button goes to https://[ZUID].manager.zesty.io/reports/metrics

Add button next to clear cache: [graph icon] All Usage

Image

### PR DESCRIPTION
**Pull request by glespinosa**
- [x] Activity Stream label
- [x] mobile responsive
- [x] mobile navbar
- [x] nav links button text
- [x] incorporate breadcrumbs to appbar
- [x] view all usage button
- [x] refactor

### CODE REVIEW

**Comment by glespinosa** [comment]
Demo mobile responsiveness and incorporate breadcrumbs in appbar

[account ui.webm](https://user-images.githubusercontent.com/44116036/194289539-ff93329f-2bee-4333-a37b-bb48b27e0d4f.webm)

Nav item box hover/item style
image
Change title of activity stream
image
View all usage
image

File: src/components/accounts/dashboard/ui/ZActivityStream.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ const ZActivityStream = ({ instancesFavorites, instances }) => {
   return (
     <>
       <Typography variant="h6" color="text.secondary">
-        Team Activity
+        Latest Actvity from your Favorite Instances
```
Comment (ardeay) [inline src/components/accounts/dashboard/ui/ZActivityStream.js]: Change text to: Latest Activity From Your Favorite Instances

File: src/components/accounts/instances/InstancesApp.js [hunk 2]
```diff
@@ -82,8 +82,8 @@ const Index = ({ children }) => {
               // borderRight: `1px solid ${grey[300]}`,
               maxWidth: { md: '384px' },
               position: 'sticky',
-              top: '60px',
-              height: `calc(100vh - 46px)`,
+              top: '114px',
+              height: `calc(100vh - 114px)`,
```
Comment (ardeay) [inline src/components/accounts/instances/InstancesApp.js]: make this number into a SCSS variable

File: src/components/accounts/profile/ProfileApp.js [hunk 3]
```diff
@@ -162,8 +162,8 @@ const Index = ({ children }) => {
               // borderRight: `1px solid ${grey[300]}`,
               maxWidth: { md: '384px' },
               position: 'sticky',
-              top: '60px',
-              height: `calc(100vh - 46px)`,
+              top: '114px',
```
Comment (ardeay) [inline src/components/accounts/profile/ProfileApp.js]: use new variable

File: src/layouts/Main/components/AppNavigation/AppNavigation.js [hunk 4]
```diff
@@ -41,6 +57,40 @@ const navigationLinks = [
...
+    id: 'preferences',
+    url: '/profile/preferences/',
+  },
+  {
+    title: 'Create Instance',
+    id: 'createInstance',
+    url: 'https://accounts.zesty.io/instances/create',
+  },
+  {
+    title: 'Legacy Account',
```
Comment (ardeay) [inline src/layouts/Main/components/AppNavigation/AppNavigation.js]: Legacy Accounts

**Review by ardeay — CHANGES_REQUESTED** [review]

**Review by ardeay — APPROVED** [review]

**Pull request merged**